recipes/conversation_watsonxai/tjbot.sample.toml

```diff
@@ -25,4 +25,9 @@ device = 'plughw:0,0' # plugged-in USB card 0, device 0
 device = 'plughw:2,0'
 backgroundAudioSuppression = 0.6
 microphoneRate = 44100
-microphoneChannels = 2
\ No newline at end of file
+microphoneChannels = 2
+
+# uncomment to change the pins for the LEDs
+# [Shine]
+# neopixel = 'gpioPin:18'
+# commonAnode = 'redPin:1, greenPin:6, bluePin:12'
```

Comment:
Does this actually work? We're not parsing these strings anywhere… unless this is somehow TOML magic?